### PR TITLE
fix favorites tokens with no icon

### PR DESCRIPTION
**Pull request by greg-schrammel**
Fixes APP-####

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -338,7 +339,13 @@ function Root() {
   return initializing ? null : (
     <ReduxProvider store={store}>
       <RecoilRoot>
-        <PersistQueryClientProvider client={queryClient} persistOptions={persistOptions}>
+        <PersistQueryClientProvider
+          client={queryClient}
+          persistOptions={persistOptions}
+          onSuccess={() => {
+            prefetchDefaultFavorites();
+          }}
```
Comment (greg-schrammel) [inline src/App.js:347]: run this after hydrating the cache, so we can skip this fetch if there is data stored already

File: src/resources/favorites.ts [hunk 2]
```diff
@@ -13,78 +13,9 @@ import { externalTokenQueryKey, fetchExternalToken } from './assets/externalAsse
 
 export const favoritesQueryKey = createQueryKey('favorites', {}, { persisterVersion: 2 });
 
-const getUniqueId = (address: AddressOrEth, chainId: ChainId) => getStandardizedUniqueIdWorklet({ address, chainId });
+const DEFAULT_FAVORITES = [DAI_ADDRESS, ETH_ADDRESS, SOCKS_ADDRESS, WBTC_ADDRESS];
```
Comment (greg-schrammel) [inline src/resources/favorites.ts:16]: replaced the predefined DEFAULT map, with lacking info, for prefetching the default tokens, only once, all subsequent app loads will grab from the persisted cache

File: src/resources/favorites.ts [hunk 3]
```diff
@@ -13,78 +13,9 @@ import { externalTokenQueryKey, fetchExternalToken } from './assets/externalAsse
 
 export const favoritesQueryKey = createQueryKey('favorites', {}, { persisterVersion: 2 });
```
Comment (greg-schrammel) [inline src/resources/favorites.ts:14]: we can add to the persister version and create a new migration, or just wait as all token icons should show up after 24hrs as the stale time will trigger a refresh

File: src/resources/favorites.ts [hunk 4]
```diff
@@ -151,7 +82,10 @@ async function fetchMetadata(addresses: string[], chainId = ChainId.mainnet) {
  * Refreshes the metadata associated with all favorites.
  */
 export async function refreshFavorites() {
-  const favorites = queryClient.getQueryData<Record<UniqueId, RainbowToken>>(favoritesQueryKey) ?? DEFAULT;
+  console.log('REFRESH');
```
Comment (walmat) [inline src/resources/favorites.ts:85]:
```suggestion
```

**Review by ibrahimtaveras00 — APPROVED** [review]
Fixes look good on both OS's ✅

**Review by walmat — APPROVED** [review]

**Review by jinchung — COMMENTED** [review]
🌮

**Pull request merged**